cmd/snap/cmd_snap_op.go

```diff
@@ -316,8 +339,14 @@ func showDone(cli *client.Client, names []string, op string, esc *escapes) error
 			fmt.Fprintf(Stdout, "internal error: unknown op %q", op)
 		}
 		if snap.TrackingChannel != snap.Channel && snap.Channel != "" {
-			// TRANSLATORS: first %s is a channel name, following %s is a snap name, last %s is a channel name again.
-			fmt.Fprintf(Stdout, i18n.G("Channel %s for %s is closed; temporarily forwarding to %s.\n"), snap.TrackingChannel, snap.Name, snap.Channel)
+			sameRisk, err := isSameRisk(snap.TrackingChannel, snap.Channel)
+			if err != nil {
```

Comment:
given how the result is used I would probably ignore the error here (also because something would have likely failed before):

if err != nil && !sameRisk ...